README.rst

```diff
@@ -185,11 +185,12 @@ For more information on archiving, see the
 Samples
 -------
 
-There are two sample applications included in this repository. To get going as fast as possible, clone the whole
+There are three sample applications included in this repository. To get going as fast as possible, clone the whole
 repository and follow the Walkthroughs:
 
 - `HelloWorld <sample/HelloWorld/README.md>`_
 - `Archiving <sample/Archiving/README.md>`_
+- `Cloud API <sample/Cloud/README.md>`_
```

Comment:
The main README.rst file includes overview docs on using the different methods ("Creating Sessions", "Generating Tokens," "Working with Archives"). We may remove this content and link to the content at https://tokbox.com/developer, but if we do not, we should add content related to these new API additions.